lib/realtime/monitoring/prom_ex/plugins/tenants.ex

```diff
@@ -2,17 +2,40 @@ defmodule Realtime.PromEx.Plugins.Tenants do
   @moduledoc false
 
   use PromEx.Plugin
+
+  alias PromEx.MetricTypes.Event
+
   require Logger
 
   @event_connected [:prom_ex, :plugin, :realtime, :tenants, :connected]
 
+  @impl true
+  def event_metrics(opts) do
+    [rpc_metrics(opts)]
+  end
+
   @impl true
   def polling_metrics(opts) do
     poll_rate = Keyword.get(opts, :poll_rate)
 
-    [
-      metrics(poll_rate)
-    ]
+    [metrics(poll_rate)]
+  end
+
+  defp rpc_metrics(_opts) do
+    Event.build(:realtime, [
+      counter(
+        "realtime.tenants.rpc.call",
+        event_name: [:realtime, :tenants, :rpc],
```

Comment:
Did you check locally that these show up in the `/metrics` endpoint?

I think these need to match what you `Telemetry.execute`. These are the events telemetry metrics "attach" to.

You are emitting events for `[:realtime, :tenants, :rpc, :calls]` but also `[:realtime, :tenants, :erpc, :calls]` those which I don't think will get picked up by PromEx currently.

Reply:
I guess we could have tests for these!

Reply:
Oh I think you just updated your PR.

Reply:
I fixed some tests since they used rpc calls and timeouts were still a pos_integer and now they are a keyword list option

indeed we could add some integrations tests to check what the metrics endpoint is returning 🤔